dds_registration/views/payment_utils.py

```diff
@@ -16,7 +16,7 @@ def invoice_download(request: HttpRequest, payment_id: int) -> HttpResponse:
         raise PermissionDenied()
 
     response = HttpResponse(content=bytes(payment.invoice_pdf().output()), content_type="application/pdf")
-    response["Content-Disposition"] = f'attachment; filename="DdS invoice {payment.invoice_no}.pdf"'
+    response["Content-Disposition"] = f'inline; filename="DdS invoice {payment.invoice_no}.pdf"'
```

Comment:
Can you help me understand this? I though `attachment` is more widely supported and would force a download, which is what we want?

Reply:
M.b., it's better to keep `attachment` if we don't want these invoices to be open in the browser but only saved to the disk. M.b., I'm wrong here, but I want to take a look at the document first usually, and only after that probably save it. In this approach, I have to find it on the disk and only then open it (in the browser again, in my case) to view.

If that behavior was desired, I'm returning it back.